src/main/scala/li/cil/oc/common/tileentity/Microcontroller.scala

```diff
@@ -168,12 +168,12 @@ class Microcontroller extends traits.PowerAcceptor with traits.Hub with traits.C
     if (node == plug.node) {
       api.Network.joinNewNetwork(machine.node)
       machine.node.connect(snooperNode)
+      connectComponents()
     }
     if (plug.isPrimary)
       plug.node.connect(componentNodes(plug.side.ordinal()))
     else
       componentNodes(plug.side.ordinal).remove()
-    connectComponents()
```

Comment:
What is this change for?

Reply:
Maybe I'm wrong but the method is called several times uselessly.